Admin/BaseMediaAdmin.php

```diff
@@ -114,8 +114,9 @@ public function getNewInstance()
                         $this->getRequest()->get(sprintf('%s[providerName]', $uniqid), null, true)
                     );
                 } else {
+                    $provider = $this->getRequest()->get($uniqid);
```

Comment:
What name would you propose? It really is wrong because that array is not just about the provider as I thought. When I debugged it before the content was something like this:

```
request = {Symfony\Component\HttpFoundation\ParameterBag} [1]
    parameters = {array} [2]
        s58a1...73 = {array} [3]
            providerName = "sonata.media.provider.image"
            category = "3"
            _token = "7lBh...0rc"
        _xml_http_request = "true"
```